test/unit/eHealth.locations/services/selected-location-factory.js

```diff
@@ -457,4 +452,39 @@ describe('Service: SelectedLocationFactory', function () {
       expect(location.levels.length).toBe(3);
     });
   });
+
+  describe('common behaviour', function () {
```

Comment:
we said that `with locations for Mali` could become `common behaviour`, and these tests could go inside there